README.md

```diff
@@ -45,6 +45,11 @@ hello-world-test:
   test:
     - docker run -e NODE_ENV=TEST harbur/hello-world-test node mochaTest
     - docker run -e NODE_ENV=TEST harbur/hello-world-test node karmaTest
+buildargs:
```

Comment:
This doesn't refer to build-args but it's a normal env variable on runtime. Here it makes sense to use NODE_ENV as use case.